create-custom-map-style/README.md

```diff
@@ -0,0 +1,904 @@
+
+    ```bash
+
+    $ amplify push
+
+    ✔ Successfully pulled backend environment <YOUR ENVIRONMENT NAME> from the cloud.
+
+    Current Environment: <YOUR ENVIRONMENT NAME>
+
+    | Category | Resource name                | Operation | Provider plugin   |
+    | -------- | ---------------------------- | --------- | ----------------- |
+    | Geo      | CreateCustomMapStyle         | Create    | awscloudformation |
+    | Auth     | createcustommapstyle0000000x | No Change | awscloudformation |
+    ? Are you sure you want to continue? Yes
+    ⠹ Updating resources in the cloud. This may take a few minutes...
+    ...
+    ✔ All resources are updated in the cloud
+    ```
+
+    1. This step created our Map Resource through Amplify. We will use this map to create our new style.
+
+    1. If you would like to check out how it looks currently, navigate to [AWS Location Services Console](https://console.aws.amazon.com/location/home) and check under "Maps".
+
+    1. We are using Esri Street Map as our base map style. This comprehensive street map includes highways, major roads, minor roads, railways, water features, cities, parks, landmarks, building footprints, and administrative boundaries.
+
+### React-Amplify Integration
+
+1. Now that we've set up all the Authorization infrastructure needed for our application, we can test it out with our frontend React application. To start we need to add the `aws-amplify` package to our dependencies
+
+    ```bash
+
+    npm i aws-amplify
+    ```
+
+1. Open `src/index.js` in your IDE
+
+1. Under your existing list of import statements create a new line and add the following:
+
+    ```js
+
+    ...
+    import  Amplify  from  "aws-amplify";
+    import  awsExports  from  "./aws-exports";
+    ```
+
+1. Next we'll need to configure our Amplify so above the `ReactDOM.render` line let's add the following:
+
+    ```js
+
+    ...
+    Amplify.configure(awsExports);
+    ...
+    ```
+
+ The `configure()` method is just setting the AWS resources that we want to use for our backend. It might look intimidating, but just remember this isn’t doing anything special here beside configuration.
+
+1. At this point your `src/index.js` file should something like this:
+
+    ```js
+    import  React  from  'react';
+    import  ReactDOM  from  'react-dom';
+    import  './index.css';
+    import  App  from  './App';
+    import  reportWebVitals  from  './reportWebVitals';
+    import  Amplify  from  'aws-amplify';
+    import  awsExports  from  './aws-exports';
+
+    Amplify.configure(awsExports);
+    
+    ReactDOM.render(
+      <React.StrictMode>
+        <App  />
+      </React.StrictMode>,
+      document.getElementById('root')
+    );
+
+
+    // If you want to start measuring performance in your app, pass a function
+    // to log results (for example: reportWebVitals(console.log))
+    // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+    reportWebVitals();
```

Comment:
Probably drop this (and sync `index.js` with what's in the repo)

Reply:
updated